learning/courses/quantum-chem-with-vqe/introduction.ipynb

```diff
@@ -41,7 +41,6 @@
     "**References**\n",
     "\n",
     "The following articles are referenced in the above video.\n",
-    "* [Medium blog post explaining VQE](https://towardsdatascience.com/the-variational-quantum-eigensolver-explained-adcbc9659c3a)\n",
```

Comment:
Forgot to mention, this 404'd and was removed after discussion with @christopherporter1